common/src/main/nl/jft/common/util/elo/EloExpectationLoader.java

```diff
@@ -0,0 +1,12 @@
+package nl.jft.common.util.elo;
+
+import java.util.List;
+
+/**
+ * @author Lesley
+ */
+public interface EloExpectationLoader {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Annotate the "EloExpectationLoader" interface with the @FunctionalInterface annotation (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1609)